www-cronrobot/Gemfile

```diff
@@ -64,7 +64,7 @@ group :development do
 end
 
 group :production do
-  gem 'pg'
```

Comment:
needs to have project selection here

![image](https://user-images.githubusercontent.com/8007722/132593460-593cef2b-4f42-41a0-9e6a-c5debfc80572.png)